nn_transfer/transfer.py

```diff
@@ -17,6 +17,8 @@
 KERAS_BETA_KEY = 'beta' + VAR_AFFIX
 KERAS_MOVING_MEAN_KEY = 'moving_mean' + VAR_AFFIX
 KERAS_MOVING_VARIANCE_KEY = 'moving_variance' + VAR_AFFIX
+KERAS_EPSILON = 1e-3
```

Comment:
Do you think we can get these values from Keras/PyTorch itself? That way if they decide to change it, this lib won't break

Reply:
Both are hardcoded default parameters. If you really want  to you can do
```python
import torch
import inspect
sig = insepct.signature(torch.nn.functional.batch_norm)
PYTORCH_EPSILON = sig.parameters['eps'].default
```
But then that breaks when they change the function/parameter name